apps/sms/js/activity_handler.js

```diff
@@ -74,44 +74,10 @@ var ActivityHandler = {
     }
   },
 
-  /**
-  * Finds a contact from a number or email address.
-  * Returns a promise that resolve with a contact
-  * or is rejected if not found
-  * @returns Promise that resolve to a
-  * {number: String, name: String, source: 'contacts'}
-  */
-  _findContactByTarget: function findContactByTarget(target) {
```

Comment:
I think this should work fine, I'll test various cases when you'll ask for review :p